contribute.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "feedthefox",
+    "description": "FirefoxOS Participation Hub",
+    "participate": {
+        "irc": "irc://irc.mozilla.org/#partinfra"
+    },
+    "repository": {
+        "url": "https://github.com/mozilla/feedthefox",
+        "license": "MPL2"
+    },
+    "bugs": {
+        "list": "https://github.com/mozilla/feedthefox/issues",
+        "report": "https://github.com/mozilla/feedthefox/issues"
```

Comment:
You could use the [new](https://github.com/mozilla/feedthefox/issues/new) url for reporting.